Fable.Remoting.ClientV2/Remoting.fs

```diff
@@ -98,3 +98,5 @@ type Remoting() =
         | _ ->
             failwithf "Cannot build proxy. Exepected type %s to be a valid protocol definition which is a record of functions" resolvedType.FullName
 
+    static member inline buildProxy<'t>(options: RemoteBuilderOptions) : 't =
+        Remoting.buildProxy(options, typeof<'t>)
```

Comment:
I've extracted the body of the function so that the rest of the code is not unnecessarily inlined and duplicated when using multiple remoting endpoints. Because of the inlining the extracted method has to be public too, but it should not cause any problems.

Reply:
Looks good, though in this case I don't believe inlining has much benefit so we can remove the `inline` modifier here